From a75c01c944343686b8ba3eb148d22657b02e8c41 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 24 Oct 2014 17:32:24 -0400
Subject: [lib] rhashtable: Spelling s/compuate/compute/

Message-id: <d9bafe6a45520a702183748377a8aa2e0120b48a.1414171887.git.jbenc@redhat.com>
Patchwork-id: 98435
O-Subject: [RHEL7.1 net PATCH 15/19] lib: rhashtable: Spelling s/compuate/compute/
Bugzilla: 1140661
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Hannes Frederic Sowa <hannes@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1140661
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8158749

commit 45d5acd3cdf36b968067b11b1e138948325e55ed
Author: Geert Uytterhoeven <geert@linux-m68k.org>
Date:   Fri Aug 8 17:19:14 2014 +0200

    lib: rhashtable: Spelling s/compuate/compute/

    Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
    Cc: Thomas Graf <tgraf@suug.ch>
    Signed-off-by: Jiri Kosina <jkosina@suse.cz>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/lib/rhashtable.c b/lib/rhashtable.c
index a2c7881..7a381e2 100644
--- a/lib/rhashtable.c
+++ b/lib/rhashtable.c
@@ -55,7 +55,7 @@ static u32 __hashfn(const struct rhashtable *ht, const void *key,
 
 /**
  * rhashtable_hashfn - compute hash for key of given length
- * @ht:  hash table to compuate for
+ * @ht:  hash table to compute for
  * @key: pointer to key
  * @len: length of key
  *
@@ -86,7 +86,7 @@ static u32 obj_hashfn(const struct rhashtable *ht, const void *ptr, u32 hsize)
 
 /**
  * rhashtable_obj_hashfn - compute hash for hashed object
- * @ht:  hash table to compuate for
+ * @ht:  hash table to compute for
  * @ptr: pointer to hashed object
  *
  * Computes the hash value using the hash function `hashfn` respectively
-- 
1.7.1